pageserver/src/tenant/mgr.rs

```diff
@@ -551,7 +550,22 @@ pub async fn detach_tenant(
     tenant_id: TenantId,
     detach_ignored: bool,
 ) -> Result<(), TenantStateError> {
-    detach_tenant0(conf, &TENANTS, tenant_id, detach_ignored).await
+    detach_tenant0(conf, &TENANTS, tenant_id, detach_ignored).await?;
+    let tmp_path = path_with_suffix_extension(conf.tenant_path(&tenant_id), TEMP_FILE_SUFFIX);
+    task_mgr::spawn(
+        task_mgr::BACKGROUND_RUNTIME.handle(),
+        TaskKind::MgmtRequest,
+        Some(tenant_id),
```

Comment:
Done

Reply:
Maybe one could move the None into a variable declaration and add a comment to it so that it is clearer for future readers who encounter this and wonder: "why is there no tenant id, this is something related to a specific tenant?".

Reply:
Added the comment